src/modules/Dropdown/DropdownMenu.js

```diff
@@ -32,6 +35,7 @@ DropdownMenu.propTypes = {
 
   /** Classes to add to the className. */
   className: PropTypes.string,
-}
 
-export default DropdownMenu
+  /** Will mark the menu as opened when nested in another menu */
+  opened: PropTypes.bool,
```

Comment:
Also, our props descriptions follow the SUI core descriptions when possible:

![image](https://cloud.githubusercontent.com/assets/5067638/18802119/d5f729b8-819b-11e6-9288-597bd4ea6ab3.png)